seed-data-scripts/seed/seed.js

```diff
@@ -338,7 +338,8 @@ const formatReference = (references, publicationId) => {
       },
       paragraphsArray[i]
     );
-
+    let referenceText = newReference.text.replace(/"(.*?)"/g, "$1")
```

Comment:
not sure what this is trying to match?
for
"beginning ""middle"" end" 

it will give 
beginning middle end 

we want:

beginning "middle" end

Reply:
actually seems to give:

beginning middle" end

Reply:
nope right first time, all " gone

Reply:
working now